testgrid/config/config.yaml

```diff
@@ -1990,8 +1990,40 @@ dashboards:
   - name: periodic-kubeadm-gce-1.6
     test_group_name: periodic-kubernetes-e2e-kubeadm-gce-1-6
 
+# These are the release *blocking* tests.  These provide a valid binary signal
+# to gate releases (alpha, beta, official).
+# This list is used by kubernetes/release/lib/releaselib.sh to provide no/nogo
+# signal for releases.
+# The first test/suite in the list is the primary suite for release purposes
+- name: release-1.6-blocking-hard
+  dashboard_tab:
+  - name: gce-1-6
+    test_group_name: ci-kubernetes-e2e-gce-release-1-6
+  - name: gce-serial-1-6
+    test_group_name: ci-kubernetes-e2e-gce-serial-release-1-6
+  - name: gce-slow-1-6
+    test_group_name: ci-kubernetes-e2e-gce-slow-release-1-6
+  - name: gce-reboot-1-6
+    test_group_name: ci-kubernetes-e2e-gce-reboot-release-1-6
+  - name: gce-scalability-1-6
+    test_group_name: ci-kubernetes-e2e-gce-scalability-release-1-6
+  - name: test-go-1-6
+    test_group_name: ci-kubernetes-test-go-release-1-6
+  - name: verify-1-6
+    test_group_name: ci-kubernetes-verify-release-1-6
+  - name: kubelet-1-6
+    test_group_name: ci-kubernetes-node-kubelet-1-6
+  - name: gke-serial-1-6
+    test_group_name: ci-kubernetes-e2e-gke-serial-release-1-6
+  - name: gke-1-6
+    test_group_name: ci-kubernetes-e2e-gke-release-1-6
+  - name: gke-slow-1-6
+    test_group_name: ci-kubernetes-e2e-gke-slow-release-1-6
+
+# These are the release "blocking" tests.
+# These need to be interpreted by a human
 # Matches the list on https://github.com/kubernetes/test-infra/issues/2029
-- name: release-1.6-blocking
+- name: release-1.6-blocking-soft
```

Comment:
I suppose we don't?  The "release blocking" set came about as a way for @kubernetes/kubernetes-release-managers to keep an eye on "release blocking" tests, however, that list did not take into account the existing *actual* set of "release blocking" tests defined in [releaselib.sh](https://github.com/kubernetes/release/blob/master/lib/releaselib.sh#L104).

Ideally we'd have ONE set of release-blocking tests (per release) that were 1) not flaky 2) can produce a true go/nogo gate for a release.